packages/node/package.json

```diff
@@ -30,11 +30,12 @@
     }
   },
   "dependencies": {
+    "@edge-runtime/vm": "1.1.0-beta.20",
     "@types/node": "*",
     "@vercel/build-utils": "5.0.4",
     "@vercel/node-bridge": "3.0.0",
     "@vercel/static-config": "2.0.1",
-    "edge-runtime": "1.0.1",
+    "edge-runtime": "v1.1.0-beta.20",
```

Comment:
Is the `v` allowed?

Reply:
it isn't, also the newest version is `1.1.0-beta.23`

Reply:
It's allowed, but we don't normally use it. I'll remove it and update the version.